### ISSUE TITLE
Export and import part of graph

### ISSUE THREAD

**Issue by GhostNr1** [Vahera/godot-orchestrator#203]
### Description

When you are doing tutorials it is good to be able to show what should be done in small code snippets. In code it is usually just a code snippet that the user can just copy and past into the code they are learning. I think something simular would be good to have here to. 

### Implementation ideas

I think it can be good if you can select part or all the nodes and export them to json/xml or some other format. Then when you do tutorial you can have a screen shot of the nodes but you can also have a copy of it that the user can just copy and paste into there project. 
This can also help if you want to have small snippets for different tasks like camera movement, key inputs and so on.
If you can have the CTRL+V to add the nodes that you have copied would be great. I think this need a lot of discussion to get right thought, how to handle variables, function, macros and so on.

**Comment by Naros** [comment]
This is a neat idea, though we'll have to see how best to do this.  We're currently re-evaluating the documentation system to think about not only multi-version but also multilingual support, and that could dictate the code needed to do this from a documentation perspective, so I'd prefer we delay until we have a defined foundation for documentation.

**Comment by GhostNr1** [comment]
Yes I have seen the discussion and I agree that this should be a part of the redesign of the documentation.
